action.yml

```
@@ -0,0 +1,69 @@
+name: "Create Escrow"
+description: "Launch a campaign escrow with provided parameters"
+author: "Hu-Fi"
+inputs:
+  chain_id: { required: true }
+  daily_volume_target: { required: true }
+  duration: { required: true }
+  exchange_name: { required: true }
+  exchange_oracle_address: { required: true }
+  recording_oracle_address: { required: true }
+  reputation_oracle_address: { required: true }
+  gas_warn_threshold: { required: true }
+  reward_amount: { required: true }
+  reward_token: { required: true }
+  start_delay: { required: true }
+  symbol: { required: true }
+
+outputs:
+  escrow_address:
+    description: "The address of the created escrow contract"
+    value: ${{ steps.launch.outputs.escrow_address }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set action path
+      shell: bash
+      run: echo "ACTION_PATH=$GITHUB_ACTION_PATH" >> $GITHUB_ENV
```

As per [this](https://docs.github.com/en/actions/tutorials/create-actions/create-a-composite-action) and [this](https://docs.github.com/en/actions/reference/workflows-and-actions/variables?versionId=free-pro-team%40latest&productId=actions&restPage=reference%2Cworkflows-and-actions%2Ccontexts#:~:text=GitHub%20Actions.-,GITHUB_PATH,-The%20path%20on)
```suggestion
      run: echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
      env:
        GITHUB_ACTION_PATH: ${{ github.action_path }}
```